### PR TITLE
Update postgres Docker tag to v16

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: Dockerfile-timescaledb [hunk 1]
```diff
@@ -17,7 +17,6 @@ RUN CGO_ENABLED=0 GOOS=linux go build -v -o /fly/bin/start ./cmd/start
 COPY ./bin/* /fly/bin/
 
 FROM wrouesnel/postgres_exporter:latest AS postgres_exporter
-
```
Comment (smorimoto) [inline Dockerfile-timescaledb:20]: To reduce the diff between two dockerfiles.

File: Dockerfile-timescaledb [hunk 2]
```diff
@@ -31,7 +30,7 @@ LABEL fly.pg-version=${PG_VERSION}
 LABEL fly.pg-manager=repmgr
 
 RUN apt-get update && apt-get install --no-install-recommends -y \
-    ca-certificates iproute2 postgresql-$PG_MAJOR_VERSION-repmgr curl bash dnsutils vim haproxy socat procps ssh gnupg rsync barman-cli barman cron \
```
Comment (smorimoto) [inline Dockerfile-timescaledb:34]: Same as above.

**Comment by smorimoto** [comment]
Ready for review!

**Comment by smorimoto** [comment]
The release of Timescale for pg16 is not yet available in the upstream repository. Keep another PR open.

**Comment by smorimoto** [comment]
@lubien Could you review this and #215?

**Comment by davissp14** [comment]
I'll do my best to take a look at it this week.  I apologize for the delay on this.

**Comment by smorimoto** [comment]
Thanks a lot 🙃

**Comment by smorimoto** [comment]
@davissp14 Could you check this?

**Comment by smorimoto** [comment]
Rebased to latest master.

**Comment by davissp14** [comment]
@smorimoto Were you able to test this?

I ran into a few issues when trying to run PG 16 with repmgr.

1. Repmgr 5.4.1 is required for PG16.
2. Repmgr 5.4.1 requires `libc > 2.38` and there doesn't seem to be a stable debian package for this.  

My initial thought was to use ubuntu for the base image, but this would likely lead to more collation issues for existing users since we use the same Dockerfile for all versions.

**Comment by davissp14** [comment]
Closing in favor of: https://github.com/fly-apps/postgres-flex/pull/233

**Pull request closed without merging**